crates/languages/src/python/config.toml

```diff
@@ -8,6 +8,7 @@ brackets = [
     { start = "{", end = "}", close = true, newline = true },
     { start = "[", end = "]", close = true, newline = true },
     { start = "(", end = ")", close = true, newline = true },
+    { start = "\"\"\"", end = "\"\"\"", close = true, newline = false, not_in = ["string"] },
```

Comment:
```suggestion
```

Autoclosing was recently intruduced in #20664 , so this change is not necessary.

Reply:
Reverted. Wasn't synced properly.